metadata.rb

```diff
@@ -5,7 +5,7 @@
 license 'Apache 2.0'
 description 'Installs/Configures Apache Mesos'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '2.0.0'
+version '2.1.0'
```

Comment:
Don't change the cookbook version.  We will need to update this cookbook to another major version for all the recent changes which are not backward compatible with what's in 2.x and up on supermarket.